### PR TITLE
fix none value for de industry

### PR DESCRIPTION
**Pull request by salistha-shakya**
Solve issue related to IME graph in de industry.

### CODE REVIEW

File: causing/utils.py [hunk 1]
```diff
@@ -348,10 +348,9 @@ def vecmat(mz):
 
 def render_dot(dot_str, filename):
     """render Graphviz graph from dot_str to svg using pydot"""
```
Comment (karlb) [inline causing/utils.py]: The docstring says `pydot`, which is not accurate anymore (we don't have to mention the library IMO).

File: causing/utils.py [hunk 2]
```diff
@@ -7,7 +7,6 @@
 from copy import deepcopy
 import json
 
-import pydot
```
Comment (karlb) [inline causing/utils.py:10]: If this was the last usage of pydot, we can remove it from the list of dependencies.

**Review by karlb — APPROVED** [review]
The code looks fine, but I have no idea which problem is solved by this changed.

Was the problem in #62 cause by pydot, or why don't you need the workaround anymore?

**Comment by salistha-shakya** [comment]
> Was the problem in #62 cause by pydot, or why don't you need the workaround anymore?

It solves the #63 issue caused by Graphviz unflatten. We are not using pydot anymore so, workaround approach caused by pydot can be ignored.

**Pull request merged**